templates/repo/issue/view_content.tmpl

```diff
@@ -102,21 +102,51 @@
 							<div class="field footer">
 								<div class="text right">
 									{{if and (or .HasIssuesOrPullsWritePermission .IsIssuePoster) (not .DisableStatusChange)}}
-										{{if .Issue.IsClosed}}
-											<button id="status-button" class="ui green basic button" tabindex="6" data-status="{{.locale.Tr "repo.issues.reopen_issue"}}" data-status-and-comment="{{.locale.Tr "repo.issues.reopen_comment_issue"}}" name="status" value="reopen">
-												{{.locale.Tr "repo.issues.reopen_issue"}}
-											</button>
-										{{else}}
-											{{$closeTranslationKey := "repo.issues.close"}}
+										{{$statusValue := "reopen"}}
+										{{$buttonTranslationKey := "repo.issues.close_as.reopen"}}
+										{{$buttonCommentTranslationKey := "repo.issues.comment_and_close_as.reopen"}}
+										{{if not .Issue.IsClosed}}
+											{{$statusValue = "close"}}
 											{{if .Issue.IsPull}}
-												{{$closeTranslationKey = "repo.pulls.close"}}
+												{{$buttonTranslationKey = "repo.pulls.close"}}
+												{{$buttonCommentTranslationKey = "repo.pulls.comment_and_close"}}
+											{{else}}
+												{{$buttonTranslationKey = "repo.issues.close_as.common"}}
+												{{$buttonCommentTranslationKey = "repo.issues.comment_and_close_as.common"}}
 											{{end}}
-											<button id="status-button" class="ui red basic button" tabindex="6" data-status="{{.locale.Tr $closeTranslationKey}}" data-status-and-comment="{{.locale.Tr "repo.issues.close_comment_issue"}}" name="status" value="close">
-												{{.locale.Tr $closeTranslationKey}}
-											</button>
 										{{end}}
+										<div class="ui buttons">
+											<button id="status-button"
+												class="ui button"
+												name="status"
+												value="{{$statusValue}}"
+												data-status="{{$.locale.Tr $buttonTranslationKey}}"
+												data-status-and-comment="{{$.locale.Tr $buttonCommentTranslationKey}}"
+											>
+												<span class="button-text">
+													{{$.locale.Tr $buttonTranslationKey}}
+												</span>
+											</button>
+											{{if not .Issue.IsPull}}
+												<div id="status-dropdown" class="ui dropdown icon button gt-ml-0">
```

Comment:
I think you need to follow the "devtest" page to see how to use "button with dropdown"